gator/orchestrate.py

```diff
@@ -40,30 +32,32 @@ def check_arguments(system_arguments):
     gg_arguments = arguments.parse(system_arguments)
     # Action: display the welcome message
     if gg_arguments.nowelcome is not True:
-        actions.append([DISPLAY, "welcome_message", VOID])
+        actions.append([DISPLAY, "welcome_message", constants.arguments.Void])
     if gg_arguments.json is True:
         # pylint: disable=global-statement
         global OUTPUT_TYPE
-        OUTPUT_TYPE = getattr(REPORT, JSON)
+        OUTPUT_TYPE = getattr(REPORT, constants.outputs.Json)
     did_verify_arguments = arguments.verify(gg_arguments)
     # arguments are incorrect
     if did_verify_arguments is False:
         # Action: display incorrect arguments message
-        actions.append([DISPLAY, "incorrect_message", VOID])
+        actions.append([DISPLAY, "incorrect_message", constants.arguments.Void])
         # Action: exit the program
-        actions.append([RUN, "run_exit", [INCORRECT_ARGUMENTS]])
+        actions.append([RUN, "run_exit", [constants.arguments.Incorrect]])
     return gg_arguments, actions
 
 
 def check_commits(system_arguments):
     """Check the commits to the git repository and return desired actions"""
     actions = []
+    # the repository is the current directory contained work to check
+    repository = constants.paths.Current_Directory
```

Comment:
I think this should probably be in the argument list below, not its own variable; this seems needlessly verbose.

Reply:
Although I am only looking at this small snippet, I am not sure what is too verbose --- referencing the Current_Directory variable? having that variable in the first place?

Reply:
Oh, I get it. You are saying that there is no need to assign it to a variable!